--- haproxy-2.6.0.ori/Makefile	2022-05-31 16:58:21.000000000 +0200
+++ haproxy-2.6.0/Makefile	2022-06-07 23:30:22.320467712 +0200
@@ -609,8 +609,6 @@
 endif
 
 ifneq ($(USE_OPENSSL),)
-SSL_INC =
-SSL_LIB =
 # OpenSSL is packaged in various forms and with various dependencies.
 # In general -lssl is enough, but on some platforms, -lcrypto may be needed,
 # reason why it's added by default. Some even need -lz, then you'll need to
